Meta/ladybird.py

```diff
@@ -38,7 +38,7 @@ def main():
         "--preset",
         required=False,
         default=os.environ.get(
-            "BUILD_PRESET", "Windows_Experimental" if platform.host_system == HostSystem.Windows else "Release"
+            "BUILD_PRESET", "Windows_CI" if platform.host_system == HostSystem.Windows else "Release"
```

Comment:
The `*_CI` presets are meant to run in CI though; shouldn't we let this reflect the setup of the Linux/macOS presets and have a sane default expanded on by _CI and _Experimental?